### ISSUE TITLE
Edit Table View: Screen Reader Instructions are not Translated

### ISSUE THREAD

**Issue by rlucke** [opencast/opencast-admin-interface#1267]
When you select an element with the keyboard, a lot of information about the operation is read out to you. Unfortunately, this information is not translated and is only available in English.
The reason for this seems to be a lack of customisation of DragDropContext. The standard text provided by the @hello-pangea/dnd library is used here. 

```
<DragDropContext
 screenReaderInstructions={{
        draggable: t(‘PREFERENCES.TABLE.DRAG_INSTRUCTIONS’),
    }}
...
```

And a corresponding text should solve the problem.

![Image](https://github.com/user-attachments/assets/7dc9f710-8e49-4093-b1cb-5db28138def8)

**Comment by Arnei** [comment]
Fixed by #1332
